Vagrantfile

```diff
@@ -464,4 +522,34 @@ Vagrant.configure(2) do |config|
     b.vm.provision "install borg", :type => :shell, :privileged => false, :inline => install_borg_no_fuse("netbsd64")
     b.vm.provision "run tests", :type => :shell, :privileged => false, :inline => run_tests("netbsd64")
   end
+
+  config.vm.define "windows10" do |b|
+    b.vm.box = "Microsoft/EdgeOnWindows10"
+    b.vm.guest = :windows
+    b.vm.boot_timeout = 600
+    b.vm.graceful_halt_timeout = 600
+
+    b.ssh.shell = "sh -l"
+    b.ssh.username = "IEUser"
+    b.ssh.password = "Passw0rd!"
+    b.ssh.insert_key = false
+
+    b.vm.provider :virtualbox do |v|
+      v.memory = 768
+      #v.gui = true
+    end
+
+    #disable unix like mount
+    b.vm.synced_folder ".", "/vagrant/borg/borg", :disabled => true
+    b.vm.synced_folder ".", "/cygdrive/c/Users/IEUser/borg", :type => "rsync", :rsync__args => ["--verbose", "--archive", "--delete", "-z"]
```

Comment:
Vagrant home is missing in this image, but I think /vagrant/borg/ could be linked to /cydrive/c/Users/IEUser using cygwin mount